cost_calculator/templates/cost_summary.html

```diff
@@ -42,7 +52,8 @@
     </thead>
   </table>
   <p>
-    <strong>Note:</strong> Nodeless cost is projected based on cpu and memory resource requests and limits for the applications running on the cluster. Projected cost uses on-demand instance price by default. Spot projections coming soon!
+    <strong>Note:</strong> Nodeless cost is projected based on cpu and memory resource requests and limits for the applications running on the cluster. Projected cost uses on-demand instance price by default. If row is yellow, it means that resource requirements are not defined for given pod.
+      This may falsify calculations and it's consider bad practice. Please set those and come back to get more accurate forecast!
```

Comment:
nit: some small grammar things
- consider -> considered
- please set those and come back to get more -> please set resource requirements for those pods to get a more